### PR TITLE
Release notes for r151044

### PR DESCRIPTION
**Pull request by citrus-it**

### CODE REVIEW

File: doc/ReleaseNotes.md [hunk 1]
```diff
@@ -2,49 +2,148 @@
...
+* The format of the OmniOS IPS repositories for this release has changed so
+  that catalogues are stored in native UTF-8 format (see
+  [pkgrepo(1)](https://man.omnios.org/pkgrepo#repository/format)). This allows
+  clients such as pkg(1) to parse the catalogues faster and with less memory
+  overhead but means that these repositories are not compatible with pkg
+  clients older than that found in the r151038 release.
+
 ### Hardware Support
 
+* PCIe hotplug is now supported for slots without a power controller.
```
Comment (oetiker) [inline doc/ReleaseNotes.md:100]:
```suggestion
* PCIe hotplug is now supported for slots without a power controller. This means, especially the hotplug support for NVME drives should work on a wider variety of motherboards then before.
```

File: doc/ReleaseNotes.md [hunk 2]
```diff
@@ -2,49 +2,148 @@
...
+
+* A new AMD Unified Memory Controller driver is available.
+
+* Additional AVX512 capabilities are now supported.
+
+* SMBIOS 3.6 support has been added.
+
+* Block devices now support GUIDs for device naming. This may cause some
+  system devices to change their device path on first boot with this release.
```
Comment (oetiker) [inline doc/ReleaseNotes.md:109]: what is the effect of this ? how will zfs cope with this ?

Comment (citrus-it) [reply doc/ReleaseNotes.md:111]: ZFS should be fine with this - and has been on all of the systems we've tested. Just a few extra messages on the console on that first boot. Rolling back to older BEs works fine too.
This is different to the other change which we've held back until r151045, which adds new device ID types and makes rollback harder.

Comment (oetiker) [reply doc/ReleaseNotes.md:111]: aah, ok, thought this was the full monty!

**Review by hadfl — APPROVED** [review]

**Pull request merged**